README.md

```diff
@@ -82,7 +82,130 @@ Create an `examples` object on each parameter that needs to go into a group in t
+  <details><summary>Sample JSON</summary>
+  
+    ```json
+    "paths": {
+      "/sample": {
+          "get": {
+            "tags": [
+              "facilities"
+            ],
+            "operationId": "getPathAndOp",
+            "parameters": [
+              {
+                "name": "page",
+                "in": "query",
+                "description": "Page of results to return per paginated response.",
+                "schema": {
+                  "type": "integer",
+                  "format": "int32",
+                  "default": 1
+                },
+                "example": 1,
+                "required": true
+              }
+            ]
+          },
+          "parameters": [
+            {
+              "name": "per_page",
+              "in": "query",
+              "description": "Number of results to return per paginated response.",
+              "schema": {
+                "type": "integer",
+                "format": "int32",
+                "default": 20
+              },
+              "example": 20,
+              "required": true
+            }
+          ]
+      }
+    }
+
+    //Eample Group
+    {
+      "name": "default",
+      "examples": {
+        "page": 1,
+        "per_page": 20
+      }
+    }
+    ```
+  </details>
+  </br> 
+
+- When the parameters set at the Operation Object level have the same unique identifier (combination of name and in values) as the Path Item Object parameters the example is pulled from the Operation Object level.
+  
+  <details><summary>Sample JSON</summary>
+    
+      ```json
+      "paths": {
+        "/sample": {
+            "get": {
+              "tags": [
+                "facilities"
+              ],
+              "operationId": "getSameValues",
+              "parameters": [
+                {
+                  "name": "page",
+                  "in": "query",
+                  "description": "Page of results to return per paginated response.",
+                  "schema": {
+                    "type": "integer",
+                    "format": "int32",
+                    "default": 1
+                  },
+                  "example": 1,
+                  "required": true
+                }
+              ]
+            },
+            "parameters": [
+              {
+                "name": "page",
+                "in": "query",
+                "description": "Number of results to return per paginated response.",
+                "schema": {
+                  "type": "integer",
+                  "format": "int32",
+                  "default": 20
+                },
+                "example": 20,
+                "required": true
+              }
+            ]
+        }
+      }
+
+      //Eample Group
+      {
+        "name": "default",
+        "examples": {
+          "page": 1
+        }
+      }
+      ```
+  </details>
+  </br>
+  
+- Using empty default example groups will result in false failures.
```

Comment:
Maybe we can provide some more information here. Maybe something like

`Using an empty default example group may result in false failures if the endpoint under test responds with an error if no parameters are provided.`

Reply:
done